bulk_insert.go

```diff
@@ -98,7 +139,19 @@ func insertObjSet(db *gorm.DB, objects []interface{}, excludeColumns ...string)
 		insertOption,
 	))
 
-	return db.Exec(mainScope.SQL, mainScope.SQLVars...).Error
+	db = db.Raw(mainScope.SQL, mainScope.SQLVars...)
```

Comment:
Why do we change from `Exec` to `Raw`?

Reply:
I find a commentary in gorm for `Raw`: 
```
// Raw use raw sql as conditions, won't run it unless invoked by other methods
//    db.Raw("SELECT name, age FROM users WHERE name = ?", 3).Scan(&result)
```
We can `Scan` or `Pluck` the result after `Raw` not `Exec`